application/modules/invoices/models/mdl_items.php

```diff
@@ -80,9 +80,9 @@ public function save($id = NULL, $db_array = NULL)
         $this->mdl_item_amounts->calculate($id);
 
         $this->load->model('invoices/mdl_invoice_amounts');
-        
-        if (isset($db_array->invoice_id)){
-            $this->mdl_invoice_amounts->calculate($db_array->invoice_id);
+
+        if (isset($db_array['invoice_id'])){
+            $this->mdl_invoice_amounts->calculate($db_array['invoice_id']);
```

Comment:
I had to reverse these changes as they break the quote and invoice saving! :warning: